src/option-parser.cc

```diff
@@ -162,107 +162,117 @@ void OptionParser::HandleArgument(size_t* arg_index, const char* arg_value) {
 
 void OptionParser::Parse(int argc, char* argv[]) {
   size_t arg_index = 0;
+  bool processing_options = true;
```

Comment:
Add a test for this in `test-option-parser.cc`

Reply:
Done